.github/workflows/ci.yml

```diff
@@ -264,6 +264,11 @@ jobs:
             runc: crun
 
     steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.10'
```

Comment:
I think when we do `sudo`, it's still using embedded go version instead of ours (so you get `compile: version "go1.14.1" does not match go tool version "go1.13.10"`). We probably need to do `sudo GOPATH=$GOPATH GOROOT=$GOROOT ...` everywhere to use our go.

Reply:
yeah, finally figured that out. Seems like because the latest Go was there before the "setup-go" action, there are caches of the standard library that get defaulted to--long-term I think it's worth reporting as that is pretty weird behavior. I'm trying `sudo -E` to see if that solves it without having to figure out how much of env. to pass through :)

Reply:
`-E` doesn't work so I'm going to have to fiddle with it until it has the right content; I think GOCACHE is probably required or something related to that as it must be looking at the cache of the host's installed Golang 1.14.x distribution?

Reply:
GOPATH+GOROOT worked for me. I'm not sure about GOCACHE